usercommandmgmt/usercommandmgmt.py

```diff
@@ -67,24 +71,20 @@ async def cc_create(self, ctx: commands.Context, command: str.lower, *, text: st
                     "Sorry, you have already created the maximum number of commands allowed by your role"
                 )
                 return
-            # regardless of the case, inform user about status of their command
-            await ctx.send("Command request submitted")
-
-            # if check is passed, the command is submitted to a specified moderation channel for evaluation
-            if not self.mod_evaluate_command(text):
-                await ctx.send(
-                    "Sorry, your requested command was deemed inappopriate by moderator"
+            # inform user about status of their command
+            if command_moderation == True:
```

Comment:
No need to compare to True, just use:
```suggestion
            if command_moderation:
```